### PR TITLE
test: ✅ add test cases to increase coverage

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     -      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/table/hooks/useFilter/FilterDropdown.tsx [hunk 1]
```diff
@@ -152,6 +152,7 @@ function FilterDropdown<RecordType>(props: FilterDropdownProps<RecordType>) {
   };
   React.useEffect(() => {
     return () => {
+      console.log(222222222);
```
Comment (xrkffgg) [inline components/table/hooks/useFilter/FilterDropdown.tsx:155]: ???

File: components/_util/scrollTo.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ interface ScrollToOptions {
 
 export default function scrollTo(y: number, options: ScrollToOptions = {}) {
   const { getContainer = () => window, callback, duration = 450 } = options;
-
+  console.log(getContainer);
```
Comment (hengkx) [inline components/_util/scrollTo.ts:16]: remove

File: components/table/Table.tsx [hunk 3]
```diff
@@ -216,8 +216,12 @@ function Table<RecordType extends object = any>(props: TableProps<RecordType>) {
     }
 
     if (scroll && scroll.scrollToFirstRowOnChange !== false && internalRefs.body.current) {
+      console.log('000');
```
Comment (hengkx) [inline components/table/Table.tsx:219]: remove

File: components/table/Table.tsx [hunk 4]
```diff
@@ -216,8 +216,12 @@ function Table<RecordType extends object = any>(props: TableProps<RecordType>) {
...
       scrollTo(0, {
-        getContainer: () => internalRefs.body.current!,
+        getContainer: () => {
+          console.log(111);
```
Comment (hengkx) [inline components/table/Table.tsx:222]: remove

**Review by hengkx — CHANGES_REQUESTED** [review]

**Review by hengkx — APPROVED** [review]

**Pull request merged**